core-tests/src/test/java/org/mule/runtime/core/internal/processor/strategy/ProactorStreamEmitterProcessingStrategyTestCase.java

```diff
@@ -506,6 +506,7 @@ public void backpressureOnInnerCpuIntensiveSchedulerBusy() throws Exception {
       flow.initialise();
       flow.start();
 
+      Thread.sleep(500);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2925))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7641&issues=AWipvBDOotIXnmsU51mr&open=AWipvBDOotIXnmsU51mr)

[ISSUE_KEY]: AWipvBDOotIXnmsU51mr